media-gfx/img2pdf/img2pdf-0.5.1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
```

Comment:
Could you also enable py3.12 here please (and run tests)?

Reply:
(see also https://bugs.gentoo.org/929621)

Reply:
Enabled 3.12, tested (3.11 & 3.12 were successful) and pushed.